launch_testing/launch_testing_examples/package.xml

```diff
@@ -13,18 +13,20 @@
   <author email="aditya.pande@openrobotics.org">Aditya Pande</author>
   <author email="sloretz@openrobotics.org">Shane Loretz</author>
 
+  <exec_depend>launch</exec_depend>
+  <exec_depend>launch_ros</exec_depend>
+  <exec_depend>launch_testing</exec_depend>
+  <exec_depend>launch_testing_ros</exec_depend>
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>rcl_interfaces</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+
   <test_depend>ament_copyright</test_depend>
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
-  <test_depend>launch</test_depend>
-  <test_depend>launch_ros</test_depend>
-  <test_depend>launch_testing</test_depend>
-  <test_depend>launch_testing_ros</test_depend>
-  <test_depend>rclpy</test_depend>
   <test_depend>ros2bag</test_depend>
   <test_depend>rosbag2_transport</test_depend>
-  <test_depend>std_msgs</test_depend>
   <test_depend>demo_nodes_cpp</test_depend>
```

Comment:
This should also be an `exec_depend`.

Reply:
Gotcha, made all the changes [496e469](https://github.com/ros2/examples/pull/357/commits/496e4693f7255eaa2303b986f4a7139989ca3d99)